sig-node/sig-node-contributor-ladder.md

```diff
@@ -0,0 +1,116 @@
+
+SIG Node differentiates code approvers and roadmap approvers by keeping the bar of enhancements approver status higher than code approver status.  In turn, achieving approver status for SIG Node is not an all or nothing experience. We require approvers to develop trust by gradually getting this status for subareas or by maintenance in adjacent projects (e.g. container runtimes).
+
+In addition to the formal requirements for the [approver role](../community-membership.md#approver), SIG Node makes these recommendations for nominees for the SIG Node top level approver status on how to demonstrate expertise and develop trust.
+
+A top-level approver MUST have intermediate approver rights in a subfolder OR adjacent vendored project or client-server flow (device plugin, container runtime, etc.).  Ideally approver rights in more than one subfolder before top-level kubelet approver is solicited is desired but not required.  This is a means of earning trust to existing approvers. 
+
+There are different ways to earn trust:
+
+**Deep expertise across multiplate sub-systems**
+
+- Demonstrated impact across multiple sub-systems.
+ - Troubleshooting complex issues that cross-subsystems with deep understanding of tooling and code base.
+ - Code contribution informed from low-level analysis or optimization of bottlnecks in kubelet.  Examples would be optimization of pod startup time, materially improving resource allocations, optimizations from pprof analysis, improvements of kubelet to kube-api traffic at scale (lease optimization, etc.).
+ - Create and merge major code simplification and/or optimization PRs indicating deep understanding of tradeoffs taken and validation of potential side effects.
+
+**Proficient in features development**
+- Drive a major KEPs at all three stages: 
+ - “alpha” - design proposal and discussions
+ - “beta” - initial customer feedback collection
+ - “GA/deprecation” - stabilizing feature, following PRRs, or managing deprecation. 
+ - Demonstrate ability to stage changes and pass PRRs keeping end users experience and Kubernetes reliability as top priorities.
+- Be a reviewer for a few major  KEPs and demonstrate meaningful participation in the review process. 
+- Drive features in connected projects which have the direct effect on SIG Node (like Runtimes - Containerd or CRI-O, cAdvisor, runc, etc.).
+- Give actionable feedback for the KEPs and initial proposals during the SIG node meetings.
+
+**Active community support**
+Be a primary PR reviewer for numerous PRs in multiple areas listed as a requirement for a reviewer.
+Actively triage issues and PRs, provide support to contributors to drive their PRs to completion.
+
+**Be present**
+Participate in SIG Node meetings by speaking about KEPs or improvements driven, or find some other way to prove the identity behind GitHub handle.
```

Comment:
```suggestion
Participate in SIG Node meetings by speaking about KEPs or improvements driven, or find some other way to engage SIG member outside of Github activities.
```

Is this closer to what you're looking for? I presume it's about engagement and not verifying identity?

Reply:
@kikisdeliveryservice it is about knowing there is a real person behind the GitHub handle.  there is some context for this request based on past requests for membership growth.  as a community, we do want to ensure we know the person by some visible engagement in a meeting, meet-up, conference, call, etc.

Reply:
gotcha, thanks for the clarification!